### PR TITLE
Fix: Random Names and Grammar

### PR DESCRIPTION
**Pull request by jlsnow301**
## About The Pull Request
Rolling strange seeds in botany made me realize how many inconsistencies there were in naming conventions, so here we are.
This PR aims to rename some reagents and fix some grammatical errors I've noticed about. There are certainly more, so I will be fixing them as I see them.

This PR now includes grammar fixes that were merged into TG: 
[#53405](https://github.com/tgstation/tgstation/pull/53405)
[#53066](https://github.com/tgstation/tgstation/pull/53066)
[#53046](https://github.com/tgstation/tgstation/pull/53046)
[#51889](https://github.com/tgstation/tgstation/pull/51889)
[#51387](https://github.com/tgstation/tgstation/pull/51387)
[#47840](https://github.com/tgstation/tgstation/pull/47840)
[#42359](https://github.com/tgstation/tgstation/pull/42359)

**Added:** It seemed illogical to display warnings (redtext) symptoms for things like alopecia, vitiligo, but only notices (bluetext) in first person. They are now both notices.
**Fixed:** All emotes were using a proc which added "." at the end, meaning messages were getting double punctuation in some cases, ex: "Frank flaps her wings ANGRILY!." This is changed to no longer have punctuation on the emote messages themselves, therefore they all end with a period. The other option is removing this message + "." functionality and manually punctuating each message.

## Why It's Good For The Game
My OCD is sated

## Changelog
:cl:
fix: Many names and descriptions are now grammatically correct.
fix: Emotes no longer add extra punctuation.
tweak: "Stimulants" renamed to amphetamine.
tweak: Viruses with purely cosmetic effects are now displayed in blue text to all parties rather than red text to observers.
/:cl:

### CODE REVIEW

File: code/modules/food_and_drinks/food/snacks_cake.dm [hunk 1]
```diff
@@ -308,16 +308,16 @@
 /obj/item/reagent_containers/food/snacks/store/cake/bscc // blackbarry strawberries chocolate cake
 	name = "blackberry and strawberry chocolate cake"
 	desc = "A chocolate cake, filled with assortment of blackberries and strawberries!"
-	icon_state = "blackbarry_strawberries_cake_coco_cake"
+	icon_state = "blackbarry_strawberries_cake_cocoa_cake"
```
Comment (ike709) [inline code/modules/food_and_drinks/food/snacks_cake.dm]: You rename icon states without editing DMIs.

Comment (jlsnow301) [reply code/modules/food_and_drinks/food/snacks_cake.dm]: Noted, changed. Also blackbarry gonna make me act up

**Comment by yorii** [comment]
Fix your changelog, it doesn't include "dramatically buffs godblood to coderbus insanity tier"

**Comment by jlsnow301** [comment]
> Fix your changelog, it doesn't include "dramatically buffs godblood to coderbus insanity tier"

Done. Realize tho that it didn't have any code to heal AT ALL (bugged?) and what's there is just copy-pasted over from earthsblood with 1 extra point added to brute, toxin, burn etc

**Comment by yorii** [comment]
It does have the code from its parent datum

**Comment by jlsnow301** [comment]
> It does have the code from its parent datum

Interesting. Didn't look so far, just remember being told it wasn't working. I will remove this aspect entirely later

**Review by zeskorion — DISMISSED** [review]
before i merge, add the change from stimulant to amphetamine to the CL so people dont get confused

**Comment by yorii** [comment]
I'm not entirely sold on the idea of renaming stimulants to amphetamine to be honest, but I'm not gonna complain too much :)

**Comment by jlsnow301** [comment]
> I'm not entirely sold on the idea of renaming stimulants to amphetamine to be honest, but I'm not gonna complain too much :)

stimulants is just very broad of a name to give. i would be okay with renaming it anything else specific - even making up a name. but just seemed abstract, "stimulants"

File: code/modules/shuttle/emergency.dm [hunk 2]
```diff
@@ -438,7 +438,7 @@
 	mode = SHUTTLE_ESCAPE
 	launch_status = ENDGAME_LAUNCHED
 	setTimer(SSshuttle.emergencyEscapeTime)
-	priority_announce("The Emergency Shuttle preparing for direct jump. Estimate [timeLeft(600)] minutes until the shuttle docks at Central Command.", null, null, "Priority")
+	priority_announce("The Emergency Shuttle preparing for direct jump. ETA: [timeLeft(600)] minutes until the shuttle docks at Central Command.", null, null, "Priority")
```
Comment (PowerfulBacon) [inline code/modules/shuttle/emergency.dm:450]: I think it sounded better befoer for this one

File: code/modules/shuttle/emergency.dm [hunk 3]
```diff
@@ -386,7 +386,7 @@
 				mode = SHUTTLE_ESCAPE
 				launch_status = ENDGAME_LAUNCHED
 				setTimer(SSshuttle.emergencyEscapeTime * engine_coeff)
-				priority_announce("The Emergency Shuttle has left the station. Estimate [timeLeft(600)] minutes until the shuttle docks at Central Command.", null, null, "Priority")
+				priority_announce("The Emergency Shuttle has left the station. ETA: [timeLeft(600)] minutes until the shuttle docks at Central Command.", null, null, "Priority")
```
Comment (PowerfulBacon) [inline code/modules/shuttle/emergency.dm]: I think this one sounded better before

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 4]
```diff
@@ -172,8 +172,8 @@
 	list_reagents = list(/datum/reagent/medicine/ephedrine = 10, /datum/reagent/consumable/coffee = 10)
 
 /obj/item/reagent_containers/hypospray/medipen/stimpack/traitor
-	desc = "A modified stimulants autoinjector for use in combat situations. Has a mild healing effect."
-	list_reagents = list(/datum/reagent/medicine/stimulants = 10, /datum/reagent/medicine/omnizine = 10)
+	desc = "A modified amphetamine autoinjector for use in combat situations. Has a mild healing effect."
+	list_reagents = list(/datum/reagent/medicine/amphetamine = 10, /datum/reagent/medicine/omnizine = 10)
```
Comment (PowerfulBacon) [inline code/modules/reagents/reagent_containers/hypospray.dm:176]: Unintentional change?

**Review by PowerfulBacon — DISMISSED** [review]

**Review by zeskorion — DISMISSED** [review]

**Comment by qwertyquerty** [comment]
conflicts

**Comment by jlsnow301** [comment]
> conflicts

resolved

File: code/modules/shuttle/emergency.dm (hunk 3 above)
Comment (jlsnow301) [reply code/modules/shuttle/emergency.dm]: Now changed to Estimate:

**Review by PowerfulBacon — DISMISSED** [review]

**Comment by PowerfulBacon** [comment]
looks through this in about 20 seconds

**Comment by PowerfulBacon** [comment]
@zeskorion review

**Comment by jlsnow301** [comment]
@ike709 can this be looked at before it hits any more merge conflicts pls

**Pull request merged**